packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -240,16 +240,20 @@ export async function loadPackageJson(path: string): Promise<any> {
 }
 
 // Helper functions for local debug correlation-id, only used for telemetry
-let localDebugCorrelationId: string | undefined = undefined;
+// Use a 2-element tuple to handle concurrent F5
+const localDebugCorrelationIds: [string, string] = ["no-session-id", "no-session-id"];
+let current = -1;
```

Comment:
:P init `current` to `0` then no need to check `current >= 0`

Reply:
Updated.